pkgs/applications/misc/go-jira/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "go-jira-0.1.14";
+
+  meta = {
+    description = "simple command line client for Atlassian's Jira service written in Go";
+    homepage = "https://github.com/Netflix-Skunkworks/go-jira";
+    licence = stdenv.lib.licenses.asl20;
+    platforms = stdenv.lib.platforms.linux;
+  };
+
+  src = fetchurl {
+    url = https://github.com/Netflix-Skunkworks/go-jira/releases/download/v0.1.14/jira-linux-amd64;
```

Comment:
Aha. Nice, I took the fast lane with the binary download, but I can do `go2nix`. I'll try to look for some examples.